tests/rest/client/v1/utils.py

```diff
@@ -184,7 +184,13 @@ def send(self, room_id, body=None, txn_id=None, tok=None, expect_code=200):
         )
 
     def send_event(
-        self, room_id, type, content={}, txn_id=None, tok=None, expect_code=200
+        self,
+        room_id,
+        type,
+        content: Optional[dict] = None,
```

Comment:
Temporary, It isn't clear if this is `dict` or `Optional[dict]`, because a `None` could've been passed through, which would've then serialised below via `json.dumps` into `null`.

Is this type correct or not? (I'll edit this snippet to adjust)

Reply:
If it can be `None` (which this obviously can due to the default value) then it should be `Optional[dict]`.

Reply:
The default value before was `{}`, bugbear bugged (heh) about this because `{}` is mutable and instantiated at compile-time, which can bring bugs, i changed it to `None` here to then do `content or {}` below, which effectively re-defaults `{}` for this parameter.

The question is; Should it? (`content or {}`), because before someone could still explicitly pass a `None` value, i'm just sanity-checking here if someone *should* pass a `None` value here at all, and so the annotation should be `dict` instead of `Optional[dict]` (because it's either the defaulted `{}` or another dict value, not and never `None`)

Reply:
I don't believe that `None` is a valid value there.